src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
Let's just use `git blame` and ask the author.
@glassez , could you comment on this, please? Why did you use `qlonglong` here?